DQMServices/Core/src/DQMStore.cc

```diff
@@ -405,14 +405,14 @@ void DQMStore::mergeAndResetMEsRunSummaryCache(uint32_t run,
 	{
 	  if(me->getTH1()->CanExtendAllAxes() && i->getTH1()->CanExtendAllAxes()) {
 	    TList list;
-	    list.Add(i->getTH1());
+	    list.Add(i->getTH1().operator->().operator->());
```

Comment:
@dmitrijus , indeed [stackoverflow agrees](https://stackoverflow.com/questions/2506793/c-life-span-of-temporary-arguments) with you:

> Temporary objects are destroyed as the last step in evaluating the full-expression that (lexically) contains the point where they were created.